src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -844,11 +853,134 @@ private static string MakeShortCategoryString(string fullCategoryName)
         #region Selection
 
         /// <summary>
-        ///     Increments the selected element by 1, unless it is the last element already
+        /// Selected member is  library search view.
+        /// </summary>
+        public NodeSearchElementViewModel CurrentlySelectedMember
+        {
+            get
+            {
+                var selectedCategory = SearchRootCategories[selectedCategoryIndex];
+                var selectedMemberGroup = selectedCategory.MemberGroups.ElementAt(selectedMemberGroupIndex);
+                var selectedMember = selectedMemberGroup.Members.ElementAt(SelectedMemberIndex);
+
+                return selectedMember;
+            }
+        }
+
+        private int selectedMemberIndex;
+        /// <summary>
+        /// Used during library search key navigation. Indicates which item index is selected.
+        /// </summary>
+        public int SelectedMemberIndex
```

Comment:
Why do we need a public property `SelectedMemberIndex` when it is not used externally?